packages/components/src/progress-bar/styles.ts

```diff
@@ -40,7 +40,7 @@ export const Track = styled.div`
 		${ COLORS.theme.foreground },
 		transparent 90%
 	);
-	border-radius: ${ CONFIG.radiusBlockUi };
+	border-radius: ${ CONFIG.radiusFull };
```

Comment:
This used to be 2px and is now 9999px. I also don't see this in the list of changes in the PR description. Did you mean to use `radiusSmall` or `radiusMedium`?

Reply:
> Since it is only 2px tall radiusXSmall (2px) is technically adequate.

I assume you meant `radiusSmall` (2px) since `radiusXSmall` is `1px`. 

Anyway, that's not a problem for your intent here, and making it rounded makes sense to me 👍